.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: php
+sudo: false
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+cache:
+  directories:
+    - node_modules
+    - vendor
+    - $HOME/phpunit-bin
+
+addons:
+  apt:
+    packages:
+      - libxml2-utils
+
+language:
+  - php
+  - node_js
+
+node_js:
+  - "6"
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+
+env:
+  - WP_VERSION=latest WP_MULTISITE=0
+  - WP_VERSION=latest WP_MULTISITE=1
+  - WP_VERSION=trunk WP_MULTISITE=0
+  - WP_VERSION=trunk WP_MULTISITE=1
+
+install:
+  - npm install
+  - export DEV_LIB_PATH=node_modules/wp-dev-lib
+  - source $DEV_LIB_PATH/travis.install.sh
+
+script:
+  - source $DEV_LIB_PATH/travis.script.sh
+
+after_script:
```

Comment:
@ThierryA, should there be a value here for `after_script`?

Reply:
Correct, it was addressed in [this commit](https://github.com/xwp/wp-amp-theme/pull/4/commits/ced132251481f2050a2ef7afefa943e831ea8ac9).